doc/source/cheatsheet/cheatsheet.qmd

```diff
@@ -100,26 +97,24 @@ import string
 import os.path
 work_dir = GetServerWorkingDirectory()
 arg_ProjectArchive = os.path.join(work_dir, "MatDesigner.wbpz")
-# Description="Upzip the archived example project file"
-Unarchive(
-    ArchivePath=arg_ProjectArchive,
+Unarchive(ArchivePath=arg_ProjectArchive,
     ProjectPath=GetAbsoluteUserPathName(work_dir + "wbpj\\MatDesigner.wbpj"),
     Overwrite=True)
     """
 wb.run_script_string(wbjn_template)
 ```
 
-Using `run_script_file()` method:
+### Using `run_script_file()` method:
 ```{python}
 wb.run_script_file("project_workflow.wbjn")
 ```
 
-You have the ability to assign any output that needs to be returned from these methods to the global variable **wb_script_result** in the script, as a JSON string.
-Workbench script returns all message summaries from the Workbench session:
+### Global variable:
+Assign necessary output from these methods to the global variable **wb_script_result** as a JSON string.
```

Comment:
```suggestion
Assign necessary output from these methods to the global variable ``wb_script_result`` as a JSON string.
```
Use code entity tag and not bold for a variable name.

Reply:
Done. Thanks!